app/views/languages/repository.erb

```diff
@@ -0,0 +1,70 @@
+<div class="container">
```

Comment:
Would you mind re-indenting this file? I don't have particularly strong feelings about HTML indentation/spacing, but this is another one of those places where some editors automatically indent, which can create noise when people submit patches.